openapi/components/schemas/ReportMonthlyRecurringRevenue.yaml

```
@@ -0,0 +1,53 @@
+type: object
+properties:
+  data:
+    type: array
+    items:
+      type: object
+      properties:
+        period:
+          description: Revenue month.
+          type: string
+          pattern: '^\d{4}-\d{2}$'
+        total:
+          description: The total MRR amount.
+          type: number
+          format: double
+        currency:
+          $ref: ./CurrencyCode.yaml
+        breakdown:
+          description: |
+            MRR could be broken down in several categories, described below.
+            The difference between current period MRR and previous period MRR would be equal to `new + reactivated + expanded - contracted - churned`.
```

nitpick: Avoid passives

`would, could, should, ...`

```suggestion
            MRR categories are described below.
            The difference between current period MRR and previous period MRR is equal to `new + reactivated + expanded - contracted - churned`.
```